VirtoCommerce.Platform.Web/Model/Diagnostics/InstalledModuleInfo.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using VirtoCommerce.Platform.Web.Model.Modularity;
+
+namespace VirtoCommerce.Platform.Web.Model.Diagnostics
+{
+    public class InstalledModuleInfo
```

Comment:
Why we need this new class? Why we can't use exists [ModuleDescriptor](https://github.com/VirtoCommerce/vc-platform/blob/master/VirtoCommerce.Platform.Web/Model/Modularity/ModuleDescriptor.cs)? Especially it has the same signature

Reply:
Yes. I agree.